Split/Network/RestClient/RestClient+TrackEvents.swift

```diff
@@ -13,7 +13,14 @@ protocol RestClientTrackEvents: RestClient {
 }
 
 extension DefaultRestClient: RestClientTrackEvents {
-  func sendTrackEvents(events: [EventDTO], completion: @escaping (DataResult<EmptyValue>) -> Void) {
-    self.execute(target: EnvironmentTargetManager.sendTrackEvents(events: events), completion: completion)
+    func sendTrackEvents(events: [EventDTO], completion: @escaping (DataResult<EmptyValue>) -> Void) {
+        do {
+            self.execute(
+                    endpoint: endpointFactory.eventsEndpoint,
+                    body: try Json.dynamicEncodeToJsonData(events),
```

Comment:
check if it is okey: we are using `dynamicEncodeToJsonData` here, but `encodeToJsonData` in RestClient+Impressions.swift, for tracking impressions (maybe tracking impressions and events should use the same JSON static method)